test-data/unit/stubgen.test

```diff
@@ -1435,3 +1435,12 @@ from typing import Any
 class C(Base, metaclass=abc.ABCMeta):
     @abstractmethod
     def other(self) -> Any: ...
+
+[case testImportErrorMessage_import]
```

Comment:
What exactly this test tests? Does it fail for the current version? (I assume it doesn't). Ideally I would add a test that checks the error formatting (or at least something that fails on master, but passes on your branch).

Reply:
Actually I need help on testing it. The error message is being print out. Any suggestion for testing it?
should I capture stdout and check it for only this test?
`
          capturedOutput = io.StringIO()
          sys.stdout = capturedOutput
`

Reply:
> should I capture stdout and check it for only this test?

I think this is OK, just don't forget to put the old value back in a `try ... finally: ...`. You can make this testing feature more generic for stubgen tests. For example you can run each test with captured output and add an additional check that stdout matches some regexp if the test case contains a `[stdout <some regexp here>]` line.

Alternatively you can simply make it a non data-driven test case similar to `mypy.teststubgen.StubgenCmdLineSuite.test_packages_found()` (this is probably even better).